source/ghost-text.js

```diff
@@ -118,7 +120,12 @@ class GhostTextField {
 				this.deactivate(false);
 				updateCount();
 			} else if (message.ready) {
-				notify('log', 'Connected! You can switch to your editor');
+				(async () => {
```

Comment:
You can make the listener on line 116 async instead

Reply:
done